openhtf/output/web_gui/src/app/plugs/user-input-plug.component.ts

```diff
@@ -65,6 +67,15 @@ export class UserInputPlugComponent extends BasePlug {
     return this.getPlugState()['text-input'];
   }
 
+  hasImage() {
+    return this.getPlugState()['show-image'];
```

Comment:
Can you just check if 'image-url' is set rather than adding another parameter?

Reply:
Fixed in a new commit